### PR TITLE
chore: typos

### PR DESCRIPTION
**Pull request by maylynn-ng**
Found a couple of typos when looking through! I've also added a bullet point to the services we provide & updated a sentence or two :)

### CODE REVIEW

File: README.md
Comment (maylynn-ng) [inline README.md:1]: mMMmmmm don't know what happened here? If it formatted it for some reason?

File: src/app/(cord)/ClientThread.tsx [hunk 1]
```diff
@@ -45,7 +45,12 @@ function CordInfo() {
             ,&nbsp;
           </li>
           <li>
-            <a href="https://docs.cord.com/rest-apis">REST APIs</a>, and{" "}
+            manage all the data for you,&nbsp;
+          </li>
+          <li>
+            <a href="https://docs.cord.com/rest-apis">REST APIs</a>, and{" "}&nbsp;
+          </li>
+          <li>
```
Comment (maylynn-ng) [inline src/app/(cord)/ClientThread.tsx:53]: not easy to see in this diff: I pulled out `webhook events` into it's own list item

File: src/app/(cord)/ClientThread.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { PagePresence, Thread } from "@cord-sdk/react";
 
 /**
  * We are adding page presence and a thread.
- * You can add more collaboration feature, see [components](https://docs.cord.com/components).
+ * You can add more collaboration features such as comments or multiple cursors! See our [components](https://docs.cord.com/components).
```
Comment (lazybean) [inline src/app/(cord)/ClientThread.tsx:7]: I do not see multiple cursors in our doc!

File: src/app/(cord)/CordIntegration.tsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ const CordIntegration = ({
         </CordProvider>
       ) : (
         <section className="cord-setup">
-          <p>Welcome to Cord&apos;s remix starter template</p>
+          <p>Welcome to Cord&apos;s Next starter template</p>
```
Comment (lazybean) [inline src/app/(cord)/CordIntegration.tsx:20]: 😅 Thank you!

File: src/app/(cord)/ClientThread.tsx [hunk 4]
```diff
@@ -45,7 +45,12 @@ function CordInfo() {
             ,&nbsp;
           </li>
           <li>
-            <a href="https://docs.cord.com/rest-apis">REST APIs</a>, and{" "}
+            manage all the data for you,&nbsp;
+          </li>
+          <li>
```
Comment (lazybean) [inline src/app/(cord)/ClientThread.tsx:50]: I think I prefer to omit that one. The copy and design of this page was given by Tom.

**Review by lazybean — COMMENTED** [review]
Thank you!

File: src/app/(cord)/ClientThread.tsx (hunk 2 above)
Comment (maylynn-ng) [reply src/app/(cord)/ClientThread.tsx:7]: whoops got ahead of myself!

File: src/app/(cord)/ClientThread.tsx (hunk 4 above)
Comment (maylynn-ng) [reply src/app/(cord)/ClientThread.tsx:50]: No worries!

**Review by lazybean — APPROVED** [review]
🚀 Thank you so much!

**Pull request merged**